java/tutorial/pom.xml

```diff
@@ -65,6 +66,21 @@
               <overWriteIfNewer>true</overWriteIfNewer>
             </configuration>
           </execution>
+
+          <execution>
+            <id>copy-dependencies-to-build</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/../../build/java</outputDirectory>
+              <overWriteReleases>false</overWriteReleases>
+              <overWriteSnapshots>false</overWriteSnapshots>
+              <overWriteIfNewer>true</overWriteIfNewer>
+            </configuration>
+          </execution>
```

Comment:
We can put this in `runtime/pom.xml`, because we only need the dependencies from the `api` and `runtime` modules.